lib/modules/aws_ip_ranges.rb

```diff
@@ -6,8 +6,13 @@ def self.cloudfront_ips
     uri = URI(PATH)
 
     begin
-      response = Net::HTTP.get(uri)
-      return parse_json_for_ips(response)
+      http_connection = Net::HTTP.new(uri.host, uri.port)
+      http_connection.read_timeout = 5
```

Comment:
is 5 enough? Should we push up to 10 instead?

Reply:
Yeah you're right, we don't have to be as short tempered with AWS on the read if a connection has been opened quickly.